pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -405,9 +407,23 @@ class Timestamp(_Timestamp):
 
     Examples
     --------
+    Using the primary calling convention:
```

Comment:
do we use this nomenclature above? if not can you reconcile

Reply:
yes in the above the doc says:
```
    Notes
    -----
    There are essentially three calling conventions for the constructor. The
    primary form accepts four parameters. They can be passed by position or
    keyword.

    The other two forms mimic the parameters from ``datetime.datetime``. They
    can be passed by either position or keyword, but not both mixed together.
```